### ISSUE TITLE
Add cross platform .ghci

### ISSUE THREAD

**Issue by lwm** [haskellpeople/hlint#21]
Currently, the `.ghci` included with Hlint has paths coded only for windows. *nix folks get a bunch of errors. Let's fix that.

**Comment by ndmitchell** [comment]
Good idea! The Windows specific aspect is purely something because I only develop on Windows.

**Comment by lwm** [comment]
FYI @module ;)
